source/max-staleness/tests/ReplicaSetWithPrimary/Secondary.json

```diff
@@ -0,0 +1,39 @@
+{
+  "heartbeatFrequencyMS": 25000,
+  "topology_description": {
+    "type": "ReplicaSetWithPrimary",
+    "servers": [
+      {
+        "address": "a:27017",
+        "type": "RSPrimary",
+        "avg_rtt_ms": 5,
+        "maxWireVersion": 5
+      },
+      {
+        "address": "b:27017",
+        "type": "RSSecondary",
+        "avg_rtt_ms": 21,
+        "maxWireVersion": 5
+      }
+    ]
+  },
+  "read_preference": {
+    "mode": "Secondary"
+  },
+  "suitable_servers": [
```

Comment:
If we're trying to test that the latency window check is applied after the suitable server check, let's make this test have two suitable servers instead of just one, where only one is in the latency window.

Reply:
resolving for re-review